README.md

```diff
@@ -39,6 +42,17 @@ Caveat: Only float values are supported. Anything else will be ignored.
 *   Topics that were matched in a subscription pattern can be hidden from the
     result through a topic configuration.
 
+*   JSON messages record each key:value pair as a unique metric, eg: the payload:
+
+        {"temperature":29.02,"linkquality":34,"humidity":55.58,"battery":100,"voltage":3005}
+
+    would expose the following metrics:
+
+        temperature{topic="zigbee2mqtt/sensor/lounge/xiaomi/WSDCGQ01LM"} 29.300000
+        linkquality{topic="zigbee2mqtt/sensor/lounge/xiaomi/WSDCGQ01LM"} 34.000000
+        humidity{topic="zigbee2mqtt/sensor/lounge/xiaomi/WSDCGQ01LM"} 54.700000
+        battery{topic="zigbee2mqtt/sensor/lounge/xiaomi/WSDCGQ01LM"} 100.000000
+        voltage{topic="zigbee2mqtt/sensor/lounge/xiaomi/WSDCGQ01LM"} 3005.000000
```

Comment:
is this actually correct? :thinking: wouldn't the key show up in the `topic` label as well, like this?

```
        temperature{topic="zigbee2mqtt/sensor/lounge/xiaomi/WSDCGQ01LM/temperature"} 29.300000
        linkquality{topic="zigbee2mqtt/sensor/lounge/xiaomi/WSDCGQ01LM/linkquality"} 34.000000
```

Due to the way the code is currently structured, I think it would have to.

Reply:
Ah yes it would be.